CHANGES.rst

```diff
@@ -55,6 +55,8 @@ Cubeviz
 - Fixed ``viewer.jdaviz_helper`` returning Specviz helper instead of Cubeviz helper after Specviz
   helper is called via ``Cubeviz.specviz``. Now ``viewer.jdaviz_helper`` always returns the Cubeviz helper. [#1546]
 
+- Increased spectral slider performance by 85% [#1550]
```

Comment:
Unless we have a benchmark set up in CI, the number could change without us knowing, so let's not be *that* specific. 😸 
```suggestion
- Increased spectral slider performance considerably. [#1550]
```